chef-utils/README.md

```diff
@@ -108,17 +108,21 @@ The OS helpers provide an alternative to comparing data from `node['os']`.
 
 Architecture Helpers allow you to determine the processor architecture of your node.
 
-* `_64_bit?`
 * `_32_bit?`
+* `_64_bit?`
+* `aarch64?`
+* `arch64?`
+* `arm64?`
```

Comment:
these helpers don't actually exist (as the specs show)

Reply:
Updated